rfc/2022/2022-07-13 - Platform Wide Documentation vs Component Specific Documentation Locations.md

```diff
@@ -0,0 +1,102 @@
+# RFC: Platform-wide VFS developer documentation will be viewed on the VA Platform Website; component-specific technical documentation will be viewed on the Console UI
+
+<!--
+The title is what you want comments on. Use the active voice in a future tense.
+Example:
+    - The website will be built using the XZY framework
+-->
+
+* Comment Deadline: `2022-08-31`
+* Team Crew and Name: [Platform/Spike Teams/Console UI](https://github.com/orgs/department-of-veterans-affairs/teams/platform-pst-console-ui)
+* Authors:
+  * [Thomas Ferrin](https://github.com/TPurple)
+  * [John Chapin](https://github.com/jchapin)
+  * [Susanna Kline](https://github.com/sdk-irl)
+* [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-architecture/pull/33)
+
+## Background
+
+The Platform currently has at least three locations for documentation:
+* The (Platform website)[https://depo-platform-documentation.scrollhelp.site/index.html], which uses (Scroll Viewport)[https://www.k15t.com/software/scroll-viewport/scroll-viewport-for-confluence-cloud] to display specific content hosted in Confluence. This website hosts the following types of content:
+    * **“Getting started”** content for onboarding VFS team members to work on the Platform
+    * **“Developer docs”** content about writing and submitting code for review
+    * **“Research and design”** content for adhering to the Platform’s guidance for research and design
+    * **“Analytics and insights”** content, such as for analytics, KPI dashboards, and monitoring
+    * **“Collaboration cycle”** documentation of the process that VFS teams are required to follow to ensure quality for all products and featured launched on [va.gov](http://va.gov/).
```

Comment:
I think this is either supposed to be "features launched" or "featured launches" but am leaning toward the first, so that's what I included in the suggestion
```suggestion
    * **“Collaboration cycle”** documentation of the process that VFS teams are required to follow to ensure quality for all products and features launched on [va.gov](http://va.gov/).
```